### ISSUE TITLE
When images are available, populate models or phenotypes pages

### ISSUE THREAD

**Issue by mellybelly** [monarch-initiative/monarch-legacy#244]
For example, this record from MGI:
http://www.informatics.jax.org/allele/key/816758
has pictures :-)

**Comment by mellybelly** [comment]
We should also pull disease images where we can get them. @pnrobinson can you advise?

**Comment by mellybelly** [comment]
(assigning to harry, please reassign as you like)

**Comment by harryhoch** [comment]
ok. Are these available from the services? If they're in the views, it shouldn't be too hard to bring them across..

**Comment by nlwashington** [comment]
image links are now available as node properties for some equivalent classes to HP ids (need to use graphson or gml to see them).  
for example HP:0010717 == EOM:01f3cc71dd4c3b9d
http://rosie.crbs.ucsd.edu:9000/scigraph/graph/neighbors/EOM_01f3cc71dd4c3b9d?depth=1&blankNodes=false&relationshipType=equivalentClass&direction=both

and you can get to the images here:
http://rosie.crbs.ucsd.edu:9000/scigraph/graph/EOM_01f3cc71dd4c3b9d.gml

**Comment by nlwashington** [comment]
images are available for the above phenotypes (from EOM), and they are also available for pathways.  these should follow the foaf:depiction links.

we could put them on a page in the jumbotron, or perhaps in a "images" tab.  (some of the phenotype images are quite unfortunate, perhaps even NSFW, so maybe they should be able to be hidden in some way)

**Comment by jmcmurry** [comment]
Agreed that some are NSFW and all might therefore be best hidden by default. If and when we implement images, we do need to have some disclaimer language in there too about suitability for viewing by youngsters.

**Comment by nlwashington** [comment]
would these be in the association tables, or just on the entity (disease/gene/phenotype/genotype/variant/etc) pages?

@kshefchek or @cmungall should we make image-\* golrs?

**Comment by kshefchek** [comment]
Is plan to link directly from EOM or serve the images from our server?  I remember some discussions about needing permission to link to their site if that is still the plan.

**Comment by jmcmurry** [comment]
I quite like the way that omim does it. They have a link to the EOM image and show it on hover.
<img width="1120" alt="screen shot 2016-09-06 at 11 34 02 am" src="https://cloud.githubusercontent.com/assets/3658863/18286047/e69aa3cc-7425-11e6-9cc1-eb03effc278f.png">
